src/desktop_notifier/common.py

```diff
@@ -289,6 +292,10 @@ class Capability(Enum):
     ATTACHMENT = auto()
     """Supports notification attachments. Allowed file types vary by platform."""
 
+    ON_CLEARED = auto()
+    """Supports distinguishing between an user closing a notification, and clearing a
+    notification programmatically, and consequently supports on-cleared callbacks"""
+
```

Comment:
How about "Supports callbacks on programmatic closing and notification timeouts"?

Reply:
Hmm… Reading through both my original suggestion five months later and yours now, I'd interpret both like "if this cap isn't set, the notification can't be closed programmatically". I made another attempt, WDYT?